src/OrchardCore.Modules/OrchardCore.Queries/Controllers/AdminController.cs

```diff
@@ -155,13 +156,17 @@ public async Task<IActionResult> CreatePost(QueriesCreateViewModel model)
             {
                 return NotFound();
             }
-
+            if (!string.IsNullOrWhiteSpace(query.Schema) && !CheckSchema(query.Schema))
+            {
+                _notifier.Error(H["Invalid schema JSON supplied."]);
+                model.Editor = await _displayManager.BuildEditorAsync(query, updater: _updateModelAccessor.ModelUpdater, isNew: true);
```

Comment:
This should add just add a `ModelState` error, and see what the result is when it is handled later in the code


```
            // If we got this far, something failed, redisplay form --> All errors should be handled by this section
            model.Editor = editor;
```

Reply:
Hi @deanmarcussen  , thanks for your help ,  `ModelState` added

Reply:
![image](https://user-images.githubusercontent.com/15613121/86513935-b4e03b80-be40-11ea-83c4-269643f84781.png)